eventOrganizer/Date.java

```diff
@@ -10,27 +21,45 @@ public class Date implements Comparable<Date> {
     private int month;
     private int day;
 
+    /**
+     * A Constructor that has a new Date object with the specified year, month, and day.
+     *
+     * @param year (YYYY)
+     * @param month The month (1-12)
+     * @param day The day of the month (1-31).
+     */
     public Date(int year, int month, int day){
         this.year = year;
         this.month = month;
         this.day = day;
     }
 
-    public boolean isValid(){
 
+    /**
+     * A public method that checks if the date is a valid calendar date.
+     *
+     * @return True if the date is valid; otherwise, false.
+     */
+    public boolean isValid(){
         if (month < 1 || month > 12){
             return false;
         }
-
         int [] daysInMonth = {0, 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
 
         if(isLeapYear(year)){
             daysInMonth[2] = 29;
         };
 
         return day >= 1 && day <= daysInMonth[month];
-
     }
+
+    /**
+     * Compares this date to another date.
+     *
+     * @param otherDate The other date to compare to.
+     * @return A negative integer if this date is before the other date,
+     *         zero if they are equal, and a positive integer if this date is after the other date.
+     */
     public int compareTo(Date otherDate) {
         if (this.year != otherDate.year) {
```

Comment:
year is a private instance variable make a getter for it and use that to get otherDate's year